.clang-format

```diff
@@ -0,0 +1,121 @@
+# copied from: https://github.com/torvalds/linux/blob/master/.clang-format
+#
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Can we include with GPL-2.0 license?

Reply:
No - we need dual license or one usable with BSD., such as MIT.